infra/docker-compose.yml

```diff
@@ -0,0 +1,241 @@
+version: '3'
+
+services:
+  cometbft-node0:
+    container_name: cometbft-node0
+    image: "cometbft/cometbft:v0.37.x"
+    ports:
+      - "26656-26657:26656-26657"
+    environment:
+      - ID=0
+      - LOG=${LOG:-cometbft-node0.log}
+      - CMT_PEX=true
+      - CMT_PROXY_APP=tcp://fendermint-node0:26658
+      - CMT_PERSISTENT_PEERS="${CMT_PERSISTENT_PEERS}"
+    volumes:
+      - ${HOME}/.ipc-testnet/node0/cometbft:/cometbft
+    healthcheck:
+      test: curl --fail http://localhost:26657 || exit 1
+      interval: 8s
+      timeout: 10s
+      retries: 20
+    networks:
+      localnet:
+        ipv4_address: 192.167.10.2
+
+  cometbft-node1:
```

Comment:
I wonder if [extends](https://docs.docker.com/compose/multiple-compose-files/extends/) would help cut down the repetition. 

In the past what I did was to put everything that can change into the `.env` file and then keep a single `docker-compose` for the set of services I want to start (just a single set of 3 containers in this case) and tell them which `.env` file to use when I up or down services, and then I can say how many containers I want in the setup script. 

I can see how this is easier to understand, but also more cumbersome to keep in sync.